packages/libs/error-stack/src/lib.rs

```diff
@@ -430,8 +425,8 @@
 //! [`set_display_hook`]: Report::set_display_hook
 //! [`set_debug_hook`]: Report::set_debug_hook
 //!
-//! [`Error`]: std::error::Error
-//! [`Error::provide`]: std::error::Error::provide
+//! [`Error`]: core::error::Error
+//! [`Error::provide`]: core::error::Error::provide
```

Comment:
Maybe as a drive-by, you could also rename these in all other places (`anyhow`, `fmt`, `eyre`, `compat`, ...)